.github/workflows/industrial_ci_action.yml

```diff
@@ -32,6 +34,11 @@ jobs:
     name: ${{ matrix.env.IMAGE }}
     runs-on: ubuntu-latest
     steps:
+      - name: Set Source Docker Variables
+        if: contains(matrix.env.IMAGE, 'source')
+        run: |
+          echo "AFTER_SETUP_UPSTREAM_WORKSPACE_EMBED=set +u && source ~/ws_moveit/install/setup.bash && set -u" >> $GITHUB_ENV
+
```

Comment:
This becomes obsolete [with https://github.com/ros-planning/moveit2/pull/698](https://github.com/ros-planning/moveit2/pull/698/files?file-filters%5B%5D=.md&file-filters%5B%5D=.yaml&file-filters%5B%5D=No+extension#diff-02fc9ec701df5b86f83b1d0963c4dc2dc724f8709abdb167c94bdea4ca492084R35-R36)
```suggestion
```

Reply:
Oh, thanks Robert, great stuff! I will remove the `AFTER_SETUP_UPSTREAM_WORKSPACE_EMBED` bit in moveit2 main as well after that gets merged!